### PR TITLE
docs: License UCL update and changelog additions

### PR DESCRIPTION
**Pull request by lkrell**
Updates in this PR:
- Update LICENSE in repo and package to strictly UCL. Ticket GOMPS-494 , verified with Legal.
- Update changelog to include 0.1.1 hotfix.
- Added potential hotfix 0.1.2 entry for the UCL license. Can remove that entry if a hotfix is not cut.

Currently entering this PR against `master` branch. Can also provide a PR to `develop` if needed.

### CODE REVIEW

File: Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
+* License updated to [Unity Companion License (UCL)](https://unity3d.com/legal/licenses/unity_companion_license) for Unity-dependent projects. See LICENSE in package for details.
+
+## [0.1.1] - 2021-04-09
+
+v0.1.1 is a hotfix for an Early Access release for Boss Room: Small Scale Co-op Sample.
+
+### Updates
+
+* Added [Third Party Contributors](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/blob/master/third-party%20contributors.md) file listing external partner contributors.
+* Corrected information for `NetworkVariableBool`.
```
Comment (SamuelBellomo) [inline Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md:18]: not just information, there was an actual code change for this

**Review by chrispope — APPROVED** [review]
Looks good

File: Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md (hunk 1 above)
Comment (lkrell) [reply Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md:18]: Updating!

**Comment by lkrell** [comment]
@chrispope  @SamuelBellomo  updated the hotfix note for the byte to bool, and changed date for 0.1.2 hotfix to today. Just need a review/approve and you can merge. Will make sure this is reflected in docsite. Thanks!

**Review by chrispope — DISMISSED** [review]

**Review by Cosmin-B — DISMISSED** [review]

**Comment by lkrell** [comment]
Chris would like to add in the repo rename to main from master to the notes. May require another approval.

**Review by chrispope — APPROVED** [review]
Looks great

**Review by SamuelBellomo — APPROVED** [review]

**Pull request merged**